style.css

```diff
@@ -1 +1,291 @@
-/*  Add your styling here */
+@font-face {
+  font-family: Questrial-Regular;
+  src: url(img/OFL.txt);
+}
+
+*:root {
+  --primary: #f2f5f8;
+  --secondary: #f56ad7;
+}
+
+body,
+a,
+ul,
+li,
+p {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  font-family: "Raleway";
+  font-size: 22px;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+.homepage-main {
+  overflow-x: hidden;
+}
+
+li {
+  list-style-type: none;
+  display: inline;
+  padding: 20px;
+  justify-content: end;
+}
+
+ul a {
+  text-decoration-color: none;
+}
+
+.button {
+  background: var(--secondary);
+  border: 2px solid var(--secondary);
+  border-radius: 2px;
+  color: var(--primary);
+  padding: 0.5em;
+}
+
+.button:hover {
+  background-color: whitesmoke;
+  color: var(--secondary);
+}
+
+.btn-tab {
+  margin: 1em;
+}
+
+.message,
+image {
+  grid-column: 3/4;
+}
+
+/*mobile*/
+.homepage-main {
```

Comment:
Well done on attempting to use the grid. Although I would only use the grid for the main content (main image, welcome, and 4 small images) and leave the others without grid since they are located on top of each other. This makes your layout to be simpler